google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/prepare/datacatalog_tag_factory.py

```diff
@@ -14,16 +14,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from google.cloud import datacatalog
+from datetime import datetime
 
+from google.cloud import datacatalog
 from google.datacatalog_connectors.commons import prepare
 
 
 class DataCatalogTagFactory(prepare.BaseTagFactory):
+    # The incoming timestamp format is UTC
+    __INCOMING_TIMESTAMP_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
If you rename the constant you can remove the comment above.
```suggestion
    __INCOMING_TIMESTAMP_UTC_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
```